timemachine/fe/rbfe.py

```diff
@@ -424,27 +463,22 @@ def estimate_free_energy_given_initial_states(initial_states, protocol, temperat
     ukln_by_lambda_by_component = np.array(ukln_by_component_by_lambda).swapaxes(0, 1)
 
     lambdas = [s.lamb for s in initial_states]
-
-    _, (ax_top, ax_btm) = plt.subplots(2, 1, figsize=(7, 9))
     overlaps_by_lambda = np.array([pair_overlap_from_ukln(u_kln) for u_kln in ukln_by_lambda_by_component.sum(axis=0)])
-    plot_overlap_summary(ax_top, ["Overall"], lambdas, [overlaps_by_lambda])
-
+    dG_errs_by_lambda_by_component = np.array(
+        [[dG_err_from_ukln(u_kln) for u_kln in ukln_by_lambda] for ukln_by_lambda in ukln_by_lambda_by_component]
```

Comment:
is this in reduced units still? (not sure where the conversion to kJ/mol is happening)

Reply:
Fixed in 6710e48